waku/v2/utils/logger.go

```diff
@@ -18,6 +19,15 @@ func Logger() *zap.Logger {
 	return log
 }
 
+// MessagesLogger returns a logger used for debug logging of receivent/sent messages
+func MessagesLogger() *zap.Logger {
```

Comment:
I was not aware we can use this logging to control levels at each subsystem this way.
Love the idea, but maybe it would be good to have subsystem at a protocol level which gives more fine-grain control. We can have msgLogging enabled either at relay or filter separately.

This function can take a prefix as an argument which each protocol can specify during initialization.

Reply:
Not sure if it is practically possible to maintain same logs across implementations. But maybe that could be done for specific trace logging like this one for messages. 


Yup, that way log level for each protocol can be changed/set to facilitate protocol specific debugging.

Reply:
Pushed a change adding prefix to message loggers.